README.md

```diff
@@ -215,6 +215,51 @@ The pre-commit hooks are run automatically by `git`. They can also be run intera
    ![Sankey diagrame:](docs/Sankey_diagram.png)
 
 
+6. supabase_db_interactions.py
+
+   #### Overview
+   This script demonstrates how to interact with a Supabase database using Python.
+   It utilises the `dotenv` library to load environment variables
+   from a `.env` file and the `supabase-py` library to perform database
+   operations.
+   The script includes functions to write data to and read data from a Supabase
+   table.
+
+   #### Features
+   - Command Line Arguments:
+
+       `--logfile-path`: Specify the path to the logfile. If not provided, it defaults to `supabase/write_to_db.log` in the working directory or the git root.
```

Comment:
When would the default be in the working directory, and when would it be in the git root (directory)? please can you clarify the circumstances when each of these would occur so there are fewer surprises :)